src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -22,8 +22,8 @@ private void TestWindowApisUnix()
             // This tests the hardcoded implementations of these APIs on Unix.
             using (Process p = Process.GetCurrentProcess())
             {
-                Assert.True(p.Responding);
-                Assert.Equal(string.Empty, p.MainWindowTitle);
+                Assert.Throws<PlatformNotSupportedException>(() => p.Responding);
```

Comment:
@danmosemsft I think that throwing is better - if we one day find a way to do the same behavior on Linux we can start supporting it without any problem, if we choose empty string then we are locked down to that behavior